### PR TITLE
Overwrites PR #238

### LINKED ISSUE #237: Migrate christoffel.py from utils to symbolic module
🐞 **Problem**
The `christoffel.py` currently resides in `utils` module.

🎯 **Goal**
  - Make a new module `symbolic`, copy the `christoffel.py` into it and edit `__init__.py` in both `utils` and `symbolic` accordingly!

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ritzvik**
Fixes #237

@OVSofia 
@shreyasbapat

### CODE REVIEW

**Comment by ritzvik** [comment]
@OVSofia I'm taking the liberty to create a new PR based on your old PR, as the changing on top of the old PR would have been tedious as a lot of changes were required. I've included you as the co-author for commits.

Kindly go through each n every addition/deletion/change done with this PR, try to understand the coding style and especially go through the test cases.

**Comment by ritzvik** [comment]
In the previous PR
  - Test cases were not running due to import issues
  - `utils/kerr_utils.py` had a dependency on `utils/bl_coord_transforms.py`. It was not taken care of before removing `utils/bl_coord_transforms.py`. Now, this error would have gone unnoticed if tests were not running. By this, I intend to emphasize the importance of tests
  - Sphinx Docs were not updated according to the changed/updated API
  - New tests were not created for `symbolic` module.
  - Old tests were not removed as and when the corresponding `.py` file was removed.

@OVSofia kindly take these are constructive criticism and try to get them right in subsequent PRs
BTW, kudos for your efforts!!

Thanks @OVSofia

**Comment by OVSofia** [comment]
Okay @ritzvik. I'm going to keep all this in mind for the next time.
But about this "Sphinx Docs were not updated according to the changed/updated API " I dont know what should I do in order to don't make the same mistake again. And neither for solve this "Old tests were not removed as and when the corresponding .py file was removed." :/  Is it use some command or something else? 

Thank you very much for the advices :)

**Comment by ritzvik** [comment]
> Okay @ritzvik. I'm going to keep all this in mind for the next time.
> But about this "Sphinx Docs were not updated according to the changed/updated API " I dont know what should I do in order to don't make the same mistake again. And neither for solve this "Old tests were not removed as and when the corresponding .py file was removed." :/ Is it use some command or something else?
> 
> Thank you very much for the advices :)

Heya! @OVSofia 
  - Whenever one adds/removes a `.py` file he/she makes the corresponding changes in `einsteinpy/docs/source` to get the changed documentation rendered. There are many files there, see one of them for example.
  - When a `.py` file is removed, the corresponding test file has to be removed from `einsteinpy/tests`. Again see one of them as example.

What I suggest is you can read each and every commit of this PR thoroughly so that you can understand how tests, documentation and module files are interlinked!

Thanks @OVSofia

**Comment by ritzvik** [comment]
Now that the basic things are in place, from next task onwards, things would not be this complicated.

File: docs/source/conf.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
     "astropy": ("http://docs.astropy.org/en/stable/", None),
     "numpy": ("https://docs.scipy.org/doc/numpy/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
-    "matplotlib": ("http://matplotlib.org", None),
-    "sympy": ("https://docs.sympy.org", None),
+    "matplotlib": ("https://matplotlib.org", None),
+    "sympy": ("https://docs.sympy.org/latest", None),
```
Comment (shreyasbapat) [inline docs/source/conf.py]: I don't see why this was required

Comment (ritzvik) [reply docs/source/conf.py]: I noticed `http` was raising a warning in the RTD build process :)

**Comment by OVSofia** [comment]
Hello :) I am working in the Metric Tensor Class from my local. I hope to have it ready for tomorrow night

**Pull request merged**